lib/flame.dart

```diff
@@ -53,24 +51,5 @@ class Flame {
 
     _bundle = bundle;
   }
-
-  /// TODO verify if this is still needed (I don't think so)
-  static void initializeWidget() {
```

Comment:
I think this is useful still, tbh I didn't even remember that we had a init method, I usually always do this `WidgetsFlutterBinding.ensureInitialized();` manually.

Reply:
Alright, should I remove the TODO comment?

Reply:
From the comment for ensureInitialized:
```
  /// You only need to call this method if you need the binding to be
  /// initialized before calling [runApp].
```
Do we need the binding initialized before runApp?

Reply:
yeah, the ensureInitialized is needed when you need to use some APIs to make full screen or things like that before the runApp.

I think we could remove the TODO

Reply:
Done